devoxx-2018-pl/exhibition/src/main/resources/tweetwallConfig.json

```diff
@@ -0,0 +1,45 @@
+{
+    "twitter": {
+        "extendedMode": true
+    },
+    "tweetwall": {
+        "query": "#devoxx",
+        "title": "The JavaFX Tweetwall for Devoxx 2017!",
```

Comment:
nitpick: title should propably be Devoxx PL 2018

Reply:
will need to update the twitter query anyway